setup/common.sh

```diff
@@ -17,3 +17,10 @@ sudo pip install vcstool
 
 # Install dependencies
 vcs import < igvc.deps
+
+
+# Install library dependencies
+sudo apt install libspatialindex-dev
+
+# Install python dependencies
+pip install -r requirements.txt
```

Comment:
This PR does add unneeded dependencies and setup but these are probably good for the future anyways.